packages/Monorepo/README.md

```diff
@@ -4,43 +4,82 @@
 [![Downloads](https://img.shields.io/packagist/dt/symplify/monorepo.svg?style=flat-square)](https://packagist.org/packages/symplify/monorepo)
 [![Subscribe](https://img.shields.io/badge/subscribe-to--releases-green.svg?style=flat-square)](https://libraries.io/packagist/symplify%2Fmonorepo)
 
+## Requirements
 
-## Install
-
-```php
-composer require symplify/monorepo
-```
+- [dflydev/git-subsplit](https://github.com/dflydev/git-subsplit) for `git subsplit` command
 
 ## Usage
 
-### 3 Steps to Build Monolitic Repository from Many Repositories
+### Build Monolithic Repository from Many Repositories
+
+- Do you have **many packages with long git history**?
+- Do you want to **turn them into monorepo**?
+- Do you want **keep their history**?
+
+That's exactly what `build` command does.
+
+#### Directories to work With
+
+You're working with 2 directories:
+
+- **monorepo directory** - monorepo will be created there, it must be empty
+- **build directory** - where you have `symplify/monorepo` installed, e.g.
 
+    ```bash
+    composer require symplify/monorepo
+    ```
 
-1. Add `monorepo.yml` with `build` section
+Do all following steps in **build directory**.
+
+#### 3 Steps to Build Monorepo
+
+1. Create `monorepo.yml` with `build` section
 
 ```yml
 parameters:
     build:
-        # link to remote git repository => local subdirectory in monorepo 
+        # remote git repository => directory in monorepo to place the package to
         'git@github.com:shopsys/product-feed-zbozi.git': 'packages/ProductFeedZbozi'
         'git@github.com:shopsys/product-feed-heureka.git': 'packages/ProductFeedHeureka'
 ```
 
-2. Prepare empty directory where, do you want to create your monorepo, e.g. `new-monorepo`. It should be outside current working directory.
+2. Run `build` command with **monorepo directory** as argument
 
-3. Run `build` command with path as argument.
+Remember, it must be outside this directory and must be empty.
 
 ```bash
-vendor/bin/monorepo build ../new-monorepo 
+vendor/bin/monorepo build ../new-monorepo
 ```
 
-And that's it.
-
-Now your packages will be in:
+3. A new `/new-monorepo` directory is created, with git history for all the packages
 
 ```bash
 /new-monorepo
     /packages
         /ProductFeedZbozi
         /ProductFeedHeureka
 ```
+
+#### How to Add Repository to Root Directory?
+
+Do you want to add remote repository not into the `packages/<name>` subdirectory but into the root directory?
+
+```yml
+# incorrect
```

Comment:
> Is there any way to make it more obvious that it is incorrect? (eg. make whole example grey?) 

Not really, just highlighters are supported here.

> Or maybe it would be sufficient to flip those examples - 1st show the right way and then describe what happens when the config is incorrect.

I'd expect classic story line in examples. From the worst, over not so bad, to better, to the best one.

How do you approach this?